static/app/views/replays/replays.tsx

```diff
@@ -122,4 +129,16 @@ const HeaderTitle = styled(PageHeading)`
   flex: 1;
 `;
 
+const TimeSinceWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledIconCalendarWrapper = styled('span')`
+  display: flex;
+  margin-right: ${space(1.5)};
+  position: relative;
+  top: -1px;
+`;
```

Comment:
You can style a component rather than add a wrapper:

```suggestion
const StyledIconCalendar = styled(IconCalendar)`
  display: flex;
  margin-right: ${space(1.5)};
  position: relative;
  top: -1px;
`;
```